ledger-api/grpc-definitions/com/daml/ledger/api/v1/experimental_features.proto

```diff
@@ -71,7 +71,8 @@ enum CommandDeduplicationType {
   // command submission, while all other duplicate commands are reported asynchronously via completions.
   ASYNC_AND_CONCURRENT_SYNC = 1;
   // Duplicate commands are always reported synchronously via a synchronous gRPC error on the command submission.
-  SYNC_ONLY = 2;
+  // Deprecated: Was used for participant-side command deduplication.
+  SYNC_ONLY = 2 [deprecated=true];
```

Comment:
If I'm not mistaken this has not been part of any release as of yet, would it be possible to just delete it in that case?

Reply:
Ah I see. Removing then